source/content/drupal-9.md

```diff
@@ -76,6 +76,110 @@ $node = Node::load(1);
 
+## Convert an existing Drupal 8 site to Drupal 9
+
+Steps to convert and migrate your Drupal 8 site to a new Drupal 9 instance.
+
+Prerequisites:
+- Install Terminus, Git, and Composer (see "[Before You Begin](drupal-9#before-you-begin)" above)
+- Install the [Terminus Site Clone](https://github.com/pantheon-systems/terminus-site-clone-plugin) plugin
+- Set environment variables with your Drupal 8 and Drupal 9 site names:
+
+```bash
+export D8_SITE=best-drupal8-site-ever
+export D9_SITE=best-drupal9-site-ever
+```
+
+1. Create a new Drupal 9 site using the link in the [Create a New Drupal 9 Site](drupal-9#create-a-new-drupal-9-site-with-integrated-composer) section above.
+
+2. Use Git to clone your new Drupal 9 site codebase to your local machine.
+
+3. On the Drupal 9 site, add your Drupal 8 site repo as remote. You can retrieve the D8 site's Git URL via the Dashboard, or with Terminus:
+
+```bash
+terminus connection:info $D8_SITE.dev --field=git_url
+```
+
+Add this as a remote on the D9 site:
+
+```bash
+git remote add original ssh://codeserver.dev.xxxx@codeserver.dev.xxxx.drush.in:2222/~/repository.git
+git fetch original
+```
+
+4. Copy over exported configuration from the original site. From your D9 site, run the following commands:
+
+```bash
+git checkout original/master -- sites/default/config
+git mv sites/default/config/* config/
+git commit -m "Add site configuration."
+```
+
+5. Compare your current `pantheon.yml` file with the new D9 `pantheon.upstream.yml`:
+
+```bash
+git diff original/master:pantheon.yml pantheon.upstream.yml
+```
+
+If you have customizations in your D8 site's `pantheon.yml` that you want to keep for D9 (e.g, a Quicksilver script or site-specific protected web paths), copy `pantheon.yml` over:
+
+```bash
+git checkout original/master -- pantheon.yml
+git commit -m "Update pantheon.yml."
+```
+
+6. Copy over any Quicksilver scripts referenced in `pantheon.yml`:
+
+```bash
+git checkout original/master -- private/scripts
+git commit -m "Add Quicksilver scripts."
+```
+
+7. List contrib modules and themes on your D8 site:
+
+```bash
+terminus drush $D8_SITE.dev -- pm:list --fields=name,version --no-core --status=enabled
+```
+
+Then use Composer on your D9 site to add these there:
+
+```bash
+composer require drupal/ctools:^3.4 drupal/redirect:^1.6 drupal/token:^1.7
+git add composer.*
+git commit -m "Add contrib projects."
+```
+
+8. Copy over any custom modules or themes from your D8 site:
+
+```bash
+git checkout original/master -- modules/custom themes/custom
+git commit -m "Add custom projects."
+```
+
+9. Check `settings.php` for any customizations to copy over:
+
+```bash
+# Fetch your D8 settings file.
+git show original/master:sites/default/settings.php > web/sites/default/original-settings.php
```

Comment:
Optionally, also add:
`diff -Nup web/sites/default/settings.php web/sites/default/original-settings.php`

Reply:
I'd omitted that line because it didn't return anything on my first test site. Now I see why it didn't! :) Added it back to step 9.